### ISSUE TITLE
[Trivy] - Workflow failure

### ISSUE THREAD

**Issue by miquelsi** [keycloak/keycloak#34440]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

The Trivy scan is failing often with the following error:

```
2024-10-29T00:19:04Z	FATAL	Fatal error	image scan error: scan error: scan failed: failed analysis: analyze error: pipeline error: failed to analyze layer (sha256:4b40bbc4d64d764acc004123ec955c3853b5c337cc6b71abf2bfff5a5d3d718b): post analysis error: post analysis error: Unable to initialize the Java DB: Java DB update failed: DB download error: OCI repository error: 1 error occurred:
	* GET https://ghcr.io/v2/aquasecurity/trivy-java-db/manifests/1: TOOMANYREQUESTS: retry-after: 763.863µs, allowed: 44000/minute
```

https://github.com/keycloak/keycloak/actions/runs/11564854882/job/32190845017

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

**Comment by miquelsi** [comment]
This failure is happening now and then. Isn't there anything we can do to improve it?

https://github.com/keycloak/keycloak/actions/runs/11603742414

**Comment by wojnarfilip** [comment]
The issue is happening consistently last 7 days https://github.com/keycloak/keycloak/actions/workflows/trivy-analysis.yml?query=branch%3Amain

**Comment by abstractj** [comment]
@wojnarfilip yes, on it today, but if you have investigated the issue and would like to propose something, please let me know.

**Comment by abstractj** [comment]
@miquelsi my apologies about that, but with a public holiday tomorrow, I won't be able to solve it this week. Next week, I will try to figure this out.
